Header header
int32 id
string status
float32 load
float32 memory
string stdout
string stdout_delta